lib/central_mail/utilities.rb

```diff
@@ -3,12 +3,18 @@
 module CentralMail
   module Utilities
     RETRIES = 3
+
     META_PART_NAME = 'metadata'
     DOC_PART_NAME = 'content'
     SUBMIT_DOC_PART_NAME = 'document'
     VALID_LOB = { 'CMP' => 'CMP', 'PMC' => 'PMC', 'INS' => 'INS', 'EDU' => 'EDU', 'VRE' => 'VRE', 'BVA' => 'BVA',
                   'FID' => 'FID', 'NCA' => 'NCA', 'OTH' => 'CMP' }.freeze
+
+    # Required Metadata fields
     REQUIRED_KEYS = %w[veteranFirstName veteranLastName fileNumber zipCode].freeze
+    # Supported fields that all consumers are permitted to provide
+    META_FIELD_NAMES_ALL_CONSUMERS = REQUIRED_KEYS + %w[source docType businessLine].freeze
```

Comment:
Looks good! I was surprised to see that `source` is marked as "required" in the `DocumentUploadMetadata` section of our documentation. If it's not actually required by us or the upstream systems, we should probably update our documentation to match.

Reply:
Created ticket